python/cuml/cuml/accel/_wrappers/sklearn/cluster.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cuml.cluster
+from cuml.accel.estimator_proxy import ProxyMixin
+
+__all__ = ("KMeans", "DBSCAN")
+
+
+class KMeans(ProxyMixin, cuml.cluster.KMeans):
```

Comment:
Changes in `_wrappers` are pretty mechanical changes from what we had before (switching `intercept` calls to subclassing from `ProxyMixin`).

In a follow-up PR I'll add a new composition-based `ProxyEstimator` base class, which we can incrementally apply to these now normal-looking class definitions.

Reply:
Yeah, something like that. And we can do it incrementally over a series of PRs, keeping `ProxyMixin` around until everything is ported.